### ISSUE TITLE
Hyperslicer not producing same image as imshow for xarray

### ISSUE THREAD

**Issue by ianhi** [mpl-extensions/mpl-interactions#269]
### Bug report
Seems that #169  did not actually fix #165
**Bug summary**

**Code for reproduction**

```python
import numpy as np
import xarray as xr
import matplotlib.pyplot as plt
from mpl_interactions import hyperslicer

arr= np.zeros([10, 150,200])
arr[:, 50:100, 50:150] = 1

arr_xr = xr.DataArray(arr, dims=('whatever', 'y', 'x'))

fig, axs = plt.subplots(1,2)

axs[0].imshow(arr[0])
x = np.linspace(0, 100)
axs[0].scatter(x,x)
axs[1].scatter(x,x)
ctrls = hyperslicer(arr_xr, ax=axs[1])
```

**Actual outcome**

![image](https://user-images.githubusercontent.com/10111092/231566861-cf7dd3b9-3273-43f3-ba2a-a72bff881622.png)

**Expected outcome**
images are exactly the same

**Comment by ianhi** [comment]
per https://matplotlib.org/stable/tutorials/intermediate/imshow_extent.html probably have different behavior depeneding on the origin argument

**Comment by ianhi** [comment]
indeed...

```python
import numpy as np
import xarray as xr
import matplotlib.pyplot as plt
from mpl_interactions import hyperslicer

arr= np.zeros([10, 150,200])
arr[:, 50:100, 50:150] = 1

arr_xr = xr.DataArray(arr, dims=('whatever', 'y', 'x'))

fig, axs = plt.subplots(2,2)

axs[0,0 ].imshow(arr[0], origin='upper')
axs[0,0].set_title('imshow - upper')
axs[0,1].set_title('hypeslicer - upper')
x = np.linspace(0, 100)
axs[0,0 ].scatter(x,x)
axs[0,1].scatter(x,x)
ctrls = hyperslicer(arr_xr, ax=axs[0,1], origin='upper')

axs[1,0 ].imshow(arr[0], origin='lower')
axs[1,0].set_title('imshow - lower')
x = np.linspace(0, 100)
axs[1,0 ].scatter(x,x)
axs[1,1].scatter(x,x)
axs[1,1].set_title('hyperslicer - lower')
ctrls = hyperslicer(arr_xr, ax=axs[1,1], origin='lower')
plt.tight_layout()
```


![image](https://user-images.githubusercontent.com/10111092/231568707-7a27772c-e4da-4a1a-9ad1-158d4e9636d8.png)

**Comment by ianhi** [comment]
fixed by #270
